Core/include/Acts/TrackFinding/TrackSelector.hpp

```diff
@@ -348,7 +353,7 @@ bool TrackSelector::isValidTrack(const track_proxy_t& track) const {
     return false;
   }
 
-  const CutConfig& cuts = m_cfg.getCuts(eta);
+  const Config& cuts = m_cfg.getCuts(eta);
```

Comment:
This seems to be a good API, so for the purposes of this PR, a good thing.

Would it be worth making a shortcut for the eta-independent case, that doesn't calculate or use `eta` until it is needed. This would be especially useful if there are no cuts on `eta` either, eg. for detectors where &eta; isn't a relevant parameter.

That would probably make the logic more complicated, so fine to postpone to another PR.